marshmallow/fields.py

```diff
@@ -994,6 +996,31 @@ def _deserialize(self, value, attr, data):
             self.fail('invalid')
 
 
+class Dict(Field):
+    """A dict field. Supports dicts and dict-like objects.
+
+    .. versionadded:: 2.1.0
+    """
+
+    default_error_messages = {
+        'invalid': 'Not a valid dict.'
+    }
+
+    def _validated(self, value):
+        if isinstance(value, collections.Mapping):
+            return value
+        else:
+            self.fail('invalid')
+
+    def _serialize(self, value, attr, obj):
+        if value is None:
+            return None
+        return self._validated(value)
```

Comment:
I think we can remove validation from `_serialize`. Data to be serialized is assumed to be valid.